package.json

```diff
@@ -106,6 +106,7 @@
     },
     "dependencies": {
         "@babel/runtime": "^7.12.5",
+        "borc": "^3.0.0",
```

Comment:
I had to add a cbor library in order to parse the signed transactions of older FW.

Reply:
Aren't the dependencies meant primarily for the connect outermost layer?  I think the parsing would actually happen in the connect service, so this could be among extendedDependencies instead, not "bloating" further existing web integrations that have nothing to do with Cardano